utils/land-protrusions.stanza

```diff
@@ -49,6 +49,9 @@ public pcb-struct ocdb/land-patterns/packages/LeadFillets:
   side:Double ; space on the edges of the lead
   courtyard-excess:Double ; additional area to add to the courtyard 
 
+public defn to-tuple (l:LeadFillets) -> [Double, Double, Double, Double]:
+  [toe(l), heel(l), side(l), courtyard-excess(l)]
```

Comment:
Sorry, I couldn't find it in the code or the design doc - does the courtyard-excess change with the density level of the landpattern? Usually extra space is added for easier rework:
- 0.1 for least
- 0.2 for nominal
- 0.4 for most
(that's in mm, across the whole width, so if you were offsetting to add the space for least material condition, you would offset by 0.05mm)

This is imperfect, and folks are still figuring it out. Here's context: https://www.pcblibraries.com/forum/component-placement-courtyards_topic1905.html

Reply:
Yes, the tables are encoded in `land-protrusions.stanza`.

Reply:
Ahh, I see it now. Thanks.